pilot/pkg/model/service.go

```diff
@@ -144,6 +144,16 @@ const (
 	ProtocolUnsupported Protocol = "UnsupportedProtocol"
 )
 
+// TrafficDirection defines whether traffic exists a service instance or enters a service instance
+type TrafficDirection string
+
+const (
+	// TrafficDirectionInbound indicates inbound traffic
+	TrafficDirectionInbound TrafficDirection = "inbound"
```

Comment:
this might require some thought.  cluster names have a a configurable max character length, so by using `inbound`/`outbound` instead of `in`/`out`, we are left with fewer characters to form unique cluster names

Reply:
WE have upto 180 characters. Since we got rid of labels, we should be safe don't you think?

Reply:
Even if we didn't have len limit: I  think in/out are still better.

Reply:
inbound/outbound is to allow us to run both old and new simultaneously. See the gross hack. ;)